api/src/bigquerytest/java/org/pmiops/workbench/api/DataSetControllerBQTest.java

```diff
@@ -128,13 +136,20 @@
     FreeTierBillingService.class,
     NotebooksServiceImpl.class,
     Provider.class,
+    UserMapper.class,
     WorkspaceMapperImpl.class
   })
   static class Configuration {
     @Bean
     public Clock clock() {
       return CLOCK;
     }
+
+    @Bean
+    @Qualifier(CommonConfig.DATASET_PREFIX_CODE)
+    String prefixCode() {
+      return "00000000";
```

Comment:
Whats this used for?

Reply:
it's a dataset identifier that's usually random. The test was previously spying on the (real) controller and overriding the value with all zeroes as if it were a mock.